Cards/CardBase.gd

```diff
@@ -50,62 +42,5 @@ func can_play_card(caster: Entity, target: Entity) -> bool:
 
 func on_card_play(caster: Entity, target: Entity) -> void:
```

Comment:
Looking at #27, on_card_play was changed to only have a target and no longer have a caster. Was this change missed or was it at some point changed back to have both ?

Reply:
Good catch, there is no caster in this function for now. Though that will probably change with the introduction of stats

Reply:
I did miss it, but if we're gonna use it later it should be fine to just keep it, right?

Reply:
Mmh maybe we should check where the function is used to make use don't introduce a bug by keeping it.

Reply:
I think it's only called in battler, and it's not like anything happens with the caster so it shouldn't make any bugs until we use the caster